### ISSUE TITLE
Research replacing gulp-util

### ISSUE THREAD

**Issue by cnlucas** [fecgov/fec-cms#5569]
### Summary

**What we're after:**
_gulp-util is deprecated. They encourage everyone to move away from it and to open tickets in any dependencies that are still using it. We should research replacing it and replace if possible_

https://www.npmjs.com/package/gulp-util

### Related issues

_List any relevant related issue(s)_

- #[issue_number]
- #[issue_number]

### Completion criteria

_What does the end state look like? Include any behavior, appearance, etc required for this ticket to be 'complete.'_

### Tech steps or considerations (optional)

_List any considerations the tech team should know. Additionally, any specific tech steps can be included here._

- [ ] First
- [ ] Next
- [ ] Then

### Future work

_Other future work that may be needed following this issue's completion._

**Comment by cnlucas** [comment]
This is blocked by webpack upgrade neo-async is used by webpack 4 which uses gulp-util

**Comment by jmetev1** [comment]
I read the article on Medium. And the impression I get is that by transitioning away you can have smaller packages (but only for your local repo since it's just a dev tool). But it doesn't seem like a security problem. They would like the people who make plugins to in the future not to use gulp-util. But it doesn't seem like a problem for this repo.

**Comment by rfultz** [comment]
@tmpayton Webpack has been upgraded and gulp-util is no longer using it.

Package-lock is telling me that gulp-css-urlencode-inline-svgs is still using gulp-util, but that's only called when we use `build-icons` (which is rare).

Thoughts?
